src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const FEMALE = 'f';
```

Comment:
You are declaring a constant inside component - in such way it would be re-created on each rerender

```suggestion

const FEMALE = 'f';

export const Person = ({ person }) => {
  const { name, age, sex, isMarried, partnerName } = person;

  ...
```